src/dsn.ml

```diff
@@ -1,13 +1,13 @@
 open Core
 
 type t' =
-  { uri : Uri.t
+  { uri : Uri_sexp.t
   ; public_key : string
   ; private_key : string option
   ; project_id : int }
-[@@deriving compare, sexp_of]
+[@@deriving sexp_of]
 
-type t = t' option [@@deriving compare, sexp_of]
+type t = t' option [@@deriving sexp_of]
```

Comment:
Is there a reason we need to remove deriving `compare`?

Reply:
It worked without it, but I found there's a Uri.compare we can use?  Let me try that.